pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[tool.poetry]
```

Comment:
I know that currently one has to cd in to the src code and then use pipenv to create the environment. I'm assuming that you're going to remove all the associated pipfiles, then change [these instructions](https://github.com/joshicola/hpc-client-josh/blob/gpu_changes/contributing.md) and [these instructions](https://github.com/joshicola/hpc-client-josh/blob/gpu_changes/doc/3-cluster-install.md#clone-the-cast-repo-and-set-up-a-virtual-environment) once we convert the hpc-client into a standalone package. Will users still have to install the entire repo, since they still need to perform the setup with the examples from the package?

Reply:
FWIW, kofalt was kinda indifferent between poetry and pipenv ("what more does poetry give you? seriously?" :D)